.circleci/config.yml

```diff
@@ -46,6 +48,18 @@ jobs:
           root: .
           paths:
             - login-workflow/dist
+            - login-workflow/coverage
+  coverage_report:
+        working_directory: ~/react-native-component-library
+        docker:
+            - image: circleci/node:12.9.1-browsers
+        steps:
+            - checkout
+            - attach_workspace:
+                at: .
+            - codecov/upload:
+                file: './login-workflow/coverage/clover.xml'
+                token: e4c45889-a27d-46ae-abd6-e0b15a8a7b57
```

Comment:
not a huge fan of having tokens in a publicly visible file, but if this is how we are supposed to do it, I guess it's fine